src/app/src/constants.js

```diff
@@ -1,4 +1,4 @@
-export const MAP_CENTER = [35.7596, -79.0193]; // North Carolina
+export const MAP_CENTER = [35.7938, -78.7232]; // Raleigh, North Carolina
```

Comment:
This slight nudge was so that the test submission would be visible by default. Better would be to add a `flyTo` call, but I figured that might be its own card (center, zoom level etc could be relevant).